tools/Customization/DevHome.Customization/Views/AddRepositoriesView.xaml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
+<UserControl
+    x:Class="DevHome.Customization.Views.AddRepositoriesView"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+    xmlns:ctControls="using:CommunityToolkit.WinUI.Controls" 
+    xmlns:viewmodels="using:DevHome.Customization.ViewModels"
+    mc:Ignorable="d">
+
+    <StackPanel>
+        <TextBlock 
+            x:Uid="AddRepositoriesTextBlock"
+            Style="{StaticResource SettingsSectionHeaderTextBlockStyle}"/>
+        <ctControls:SettingsCard
+            x:Uid="AddRepositoriesCard"
+            Grid.Column="1">
+            <Button 
+                x:Uid="AddFolderButton" 
+                Style="{ThemeResource AccentButtonStyle}" 
+                Command="{x:Bind ViewModel.AddFolderClickCommand}" />
+        </ctControls:SettingsCard>
+        <ItemsRepeater 
+            Name="ItemsRepeaterForAllRepoPaths" 
+            Visibility="{x:Bind ViewModel.Visibility}"
```

Comment:
Do you need to set this (and have a corresponding property) at all? If the list is empty, won't the ItemsRepeater be empty (and therefore invisible)?

Reply:
Hmmm I was thinking of a case where the user was previously using the experimental feature and decides to turn it off temporarily. At the same time, though the user keeps the File Explorer Version Control Integration setting (to be added in a future PR on the same page) enabled. In this case the list may not be empty.

Reply:
Resolved offline and committed new changes. Thanks!